frontend/src/js/util/resourceUtils.ts

```diff
@@ -72,6 +72,10 @@ export function getDefaultView(resource: Resource): string | undefined {
         return undefined;
     }
 
+    if (resource.transcript) {
+        return "transcript." + Object.keys(resource.transcript)[0];
```

Comment:
Two comments:

1) I think the custom for string interpolation (following what's already in this file - and modern practice in general since ES6) would be `transcript.${Object.keys(...)}`` rather than the `+` operator.
2) Question (this is not a blocker, it's just for my understanding): is there ever a chance where a transcript could be more than one language? Just asking because since you're always indexing into the first element of the array - and also looking at the current modeling of `transcript`, which is 
```typescript
    transcript?: {
        [lang: string]: HighlightableText
    }
```
it doesn't look like that's the case. So I wonder why it wasn't modeled like so:
```
    transcript?: {
        lang: string,
        text: HighlightableText
    }
```
That would allow you to do something like
```typescript
return `transcript.${resource.transcript.lang}`
```
which feels cleaner.
Again, not a blocker - just curious :)

Reply:
Thanks @marsavar for looking. I made a change for your 1st comment :)

Regarding the 2nd comment, yes there are cases where we have transcription on several languages. The reason I'm choosing the first index, is that this would return the view query parameter (e.g. view=transcript.arabic) which is used when the viewer page opens. The user would still be able to change the tabs to go to other transcriptions if they exist but the first one is in the view at first.